### PR TITLE
fix: modified ED check in EVM withdrawal for XCM assets

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* fix: modified ED check (native token balance in origin chain) for EVM withdrawal for XCM assets
* fix: updated `minBridgeAmount` for AUSD and KAR token because the amount should be more than ED (Acala tokens have ED in each XCM assets )
  *  ref: https://wiki.acala.network/get-started/acala-network/acala-account 
* fix: removed `MOONBEAM` option from XCM transfer UI in ASTR

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Fixes**
* fix: modified ED check (native token balance in origin chain) for EVM withdrawal for XCM assets

Account's native token balance: [0.2ACA](https://acala.subscan.io/account/24Y913oKBCBpPP3q8RLbDCBoZMxneU5VSnWHHkYVRH2cEnvq) 

=Before=

=After=

* fix: removed `MOONBEAM` option from XCM transfer UI in ASTR
=Before=

=After=

### CODE REVIEW

**Review by Kahonnohak — APPROVED** [review]
Thank you so much!LGTM

File: src/hooks/xcm/useXcmBridge.ts [hunk 1]
```diff
@@ -103,11 +102,15 @@ export function useXcmBridge(selectedToken: Ref<Asset>) {
...
+    // Todo: un-comment-out after channel between Astar and Moonbeam has been opened
+    // return destChain.value.name === Chain.MOONRIVER || destChain.value.name === Chain.MOONBEAM;
+    return destChain.value.name === Chain.MOONRIVER;
   });
 
   const isMoonbeamDeposit = computed<boolean>(() => {
-    return srcChain.value.name === Chain.MOONRIVER || srcChain.value.name === Chain.MOONBEAM;
+    // Todo: un-comment-out after channel between Astar and Moonbeam has been opened
+    // return srcChain.value.name === Chain.MOONRIVER || srcChain.value.name === Chain.MOONBEAM;
+    return srcChain.value.name === Chain.MOONRIVER;
```
Comment (hoonsubin) [inline src/hooks/xcm/useXcmBridge.ts:113]: I noticed that adding direct references to this script is not a scalable solution. We should keep the code composable by isolating chain config to different chains

Comment (impelcrypto) [reply src/hooks/xcm/useXcmBridge.ts:113]: Noted, currently, I'm working on the new transfer page and this hook is refactored too. How about let me modify in the branch of the new transfer page? Tbh, this is a bit challenging but I'll try to clean up more.

Comment (impelcrypto) [reply src/hooks/xcm/useXcmBridge.ts:113]: @hoonsubin Please help take a look at my comment when you are available 🙏

Comment (hoonsubin) [reply src/hooks/xcm/useXcmBridge.ts:113]: Then what about adding this to our backlog for later?

**Review by hoonsubin — APPROVED** [review]

**Comment by hoonsubin** [comment]
@impelcrypto RPC check is failing

**Comment by impelcrypto** [comment]
@hoonsubin  RPC check is not stable and I've reported to @niklabh. 
https://github.com/AstarNetwork/astar-apps/pull/480#issuecomment-1206068410

File: src/hooks/xcm/useXcmBridge.ts (hunk 1 above)
Comment (impelcrypto) [reply src/hooks/xcm/useXcmBridge.ts:113]: ok!

**Comment by niklabh** [comment]
Fixing rpc check. This is because default parachain-id  is abolised with new release

**Comment by niklabh** [comment]
https://github.com/AstarNetwork/astar-apps/pull/485

**Pull request merged**